contracts/v3/Financing.sol

```diff
@@ -0,0 +1,82 @@
+pragma solidity ^0.7.0;
+
+// SPDX-License-Identifier: MIT
+
+import './ModuleRegistry.sol';
+import './Proposal.sol';
+import './Voting.sol';
+import './Bank.sol';
+import '../SafeMath.sol';
+import '../ReentrancyGuard.sol';
```

Comment:
I see you've added this smart contract but you haven't used it. If you end up not using it I would remove it

Reply:
Oh, indeed. I still need to add that to the `processProposal` function.